### PR TITLE
Try adding test files

### PR DESCRIPTION
**Pull request by 8bitsam**
Added the test files. Notably, the tests use `unittest`, not sure how important it is to migrate over to `pytest`.

### CODE REVIEW

File: src/diffpy/fourigui/tests/integration_test.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import unittest
+
+import h5py
+import numpy as np
+
+from ..fourigui.fourigui import Gui
```
Comment (sbillinge) [inline src/diffpy/fourigui/tests/integration_test.py:6]: this is not right.  We want it imported as a package not by a relative path.

**Review by sbillinge — COMMENTED** [review]
please can you see inline comment.

Also, let's check how big the h5 files are before merging them here.  How big are they?

**Comment by 8bitsam** [comment]
The entire test_data folder is 66mb, the h5 files are all 8.2mb

**Comment by sbillinge** [comment]
> Added the test files. Notably, the tests use `unittest`, not sure how important it is to migrate over to `pytest`.

we will move to pytest.  But pytest can run unittest so we don't have to change any of the existing tests, but we do want to change how they are run following the cookiecutter norms.

**Comment by 8bitsam** [comment]
> > Added the test files. Notably, the tests use `unittest`, not sure how important it is to migrate over to `pytest`.
> 
> we will move to pytest. But pytest can run unittest so we don't have to change any of the existing tests, but we do want to change how they are run following the cookiecutter norms.

I changed the tests to use pytest. I think we can merge this PR and #1, see if tests pass, then make a new PR if they don't.

**Comment by bobleesj** [comment]
@8bitsam Hi Sam, I just wanted to double check with you. Before you did any cookiecutting, did pytest run locally?

**Review by sbillinge — COMMENTED** [review]
Let's close this and redo. I am nervous about putting the h5 files in here, and especially as they would then later be moved by the cookiecut.

We may want save the other work on this PR, but let's discuss in person first and make a decision.

**Comment by 8bitsam** [comment]
Closed by #6

**Pull request closed without merging**